### PR TITLE
Make README

### PR DESCRIPTION
**Pull request by tusharhero**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1 +1,18 @@
 # GNU Linux India
+
+Welcome to the "Spirit of GLI", as we endeavor to bring you a new and
+improved version of our community's activities in 2022.
```
Comment (gtlsgamr) [inline profile/README.md:4]: Change this to generalised GLI not "Spirit of GLI"

Comment (gtlsgamr) [reply profile/README.md:4]: Also 2022?

**Review by gtlsgamr — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 1 above)
Comment (tusharhero) [reply profile/README.md:4]: I guess it hallucinated that

Comment (tusharhero) [reply profile/README.md:4]: I will just change it to "the past"

**Comment by tusharhero** [comment]
@gtlsgamr done

**Pull request closed without merging**